src/main/java/com/spartronics4915/frc2020/commands/IndexerCommands.java

```diff
@@ -207,10 +201,9 @@ public LoadFromIntake(Indexer indexer)
 
             addCommands(
                 new EndLaunch(mIndexer), // for safety
-                new WaitForBallHeld(mIndexer), 
-                new LoadBallToSlot(mIndexer, 0), 
-                new Spin(mIndexer, 1),
-                new InstantCommand(() -> mIndexer.addBalls(1), mIndexer), 
+                new WaitForBallHeld(mIndexer),
+                new LoadBallToSlot(mIndexer, 0),
+                new Spin(mIndexer, 1), new InstantCommand(() -> mIndexer.addBalls(1), mIndexer),
```

Comment:
would be nice to replace the InstantCommand with a named command (like Spin, et all).  This will be helpful for diagnostic purpose.